docs/configuration.md

```diff
@@ -330,3 +330,18 @@ telemetry:
   interval: 2m0s
   enabled: true
 ```
+
+### Config validation
+
+k0s Command-Line Interface has ability to validate config syntax:
+
+```
+$ k0s validate config --config path/to/config/file
+```
+
+`validate config` sub-command can validate the following:
+
+1. YAML formatting
+2. [SANs addresses](#specapi-1)
+3. [Network providers](#specnetwork-1)
+4. [Worker profiles](#specworkerprofiles) 
```

Comment:
I would put this section above the "Full Config Reference". IMO, by adding this section to the bottom we're burying a very important feature.